pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
   # Make sure to update the mkdocs.yml file when
   # changing the version
   # (plugins.mkdocstrings.handlers.python.import)
-  "frequenz-channels >= 0.16.0, < 0.17.0",
+  "frequenz-channels >= 1.0.0b2, < 1.1.0",
```

Comment:
New betas will probably add more breaking changes, so I think for now we should pin it to `1.0.0b2` exactly (`== 1.0.0b2`).

Reply:
For some reason I doubt that this module will change that drastically in the near future. But who knows. I've adapted this line. :)

Reply:
The channels repo before the final 1.0? It will change for sure, quite drastically (module restructure and other minor API changes)

Reply:
I'm fine with that. I've change the line already. Do you mind :+1:'ing? Or am I missing anything?

Reply:
still needs release notes.  not sure if you saw Luca's comment: https://github.com/frequenz-floss/frequenz-sdk-python/pull/764#pullrequestreview-1712373078

Reply:
> The release notes should mention this dependency bump as it will force downstream users to upgrade the channels version too

oh, right, I did NOT see this. Also, I mentioned in the top body why I didn't do it. but i can do. hold on :)

Reply:
Yeah, but not all version bumps are the same. This one is pretty important and we'll have to do it starting with v1.0.0. It is true that we are still at pre-releases, so it would be fine to skip it this time. If you prefer to do that, I'd be fine with it too.

Reply:
changelog added

Reply:
Just FYI, we did include bumps on the channels repo in the release notes in the past, for example: https://github.com/frequenz-floss/frequenz-sdk-python/releases/tag/v0.23.0